src/C1T2X_OBU.py

```diff
@@ -21,16 +21,17 @@
 
 from Networking.networking import UDP_NET
 
-# LOGGING
+### Logging
 c1t2x_logger = None
-# Get Logging Level
 LOGGING_LEVEL = logging.INFO
 logs_directory = os.path.join(os.getcwd(), "Logs")
```

Comment:
Path-related operations should be update from `os` to [`pathlib`](https://docs.python.org/3/library/pathlib.html) since it is current best practice.